HLTrigger/JetMET/src/HLTFatJetMassFilter.cc

```diff
@@ -128,11 +129,11 @@ HLTFatJetMassFilter<jetType>::hltFilter(edm::Event& iEvent, const edm::EventSetu
   
   // apply radiation recovery
   for ( recojet = recojets.begin() ; recojet != recojets.end() ; recojet++) {
-    double DeltaR1 = sqrt(pow(recojet->phi()-j1.phi(), 2.)+pow(recojet->eta()-j1.eta(),2.));
-    double DeltaR2 = sqrt(pow(recojet->phi()-j2.phi(), 2.)+pow(recojet->eta()-j2.eta(),2.));
-    if(DeltaR1 < DeltaR2 && DeltaR1 < fatJetDeltaR_) {

```

Comment:
@duanders from the code, I think the bug you are trying to fix is that |phi2 - phi1| does not properly wrap around π , right ?